1.traverseLinkedList.js

```diff
@@ -19,8 +32,23 @@
 // [556, 41, 33, 150, 11]
 
 
-function traverse(list) {
+// if there is no node to start off with, return null or []
+// if there is a node, save it to the arr 
+// then keep going thru the nodes and saving them
+// oh need to set curr vari 
 
+function traverse(list) {
+    let current = list;
+    let arr = []
+    if(!list) return [];
+    if (list) {
+        arr.push(current.value)
+        // push it into the arr //(the actual value)
+        // then go to the next node and push that in //value // new head becomes current
+        current = current.next
+    }
+    // return the arr you pushed the vals into
+    return arr;
 }
 
```

Comment:
Mostly right idea but missing the loop. You need to loop to traverse lists, unless leveraging recursion.